### PR TITLE
Update korean.txt

### PR DESCRIPTION
**Pull request by telk5093**
I changed korean currency into utf-8 code, too.

### CODE REVIEW

File: data/language/korean.txt [hunk 1]
```diff
@@ -3889,6 +3889,32 @@ STR_5549    :연/월/일
...
+STR_5568    :암호 필요
+STR_5569    :이 서버에 접속하려면 암호가 필요합니다
+STR_5570    :서버 검색
+STR_5571    :게임 참여
+STR_5572    :즐겨찾기 등록
+STR_5573    :즐거찾기 제거
+STR_5574    :서버 이름:
+STR_5575    :최대 플레이어 수:
+STR_5576    :포트:
+STR_5577    :대한민국 원 (\)
```
Comment (Gymnasiast) [inline data/language/korean.txt]: This doesn't seem the correct Won sign. You can just copy and paste a real Won sign into this without problems.

Comment (telk5093) [inline data/language/korean.txt]: Usually korean uses backslash(\) as a Won sign. And I checked it is shown correctly. (using korean font)
Do you want me to change it to ₩(U+20A9) although?

Comment (Gymnasiast) [inline data/language/korean.txt]: Yes, please.

**Comment by telk5093** [comment]
I editted \ into ₩.

**Comment by Gymnasiast** [comment]
Thanks. There are some changed strings as well. See the Issues page for those.

**Pull request merged**